packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -123,18 +123,16 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     protected createInputTypes(model: ModelDefinition) {
         const modelFields = Object.values(model.graphqlType.getFields());
 
+        // TODO relationships?
         return new GraphQLInputObjectType({
-            // TODO
             name: `${model.graphqlType.name}Input`,
             fields: () => (modelFields.filter((field: GraphQLField<any, any>) => {
                 const fieldBaseType = getBaseType(field.type);
 
                 return !isObjectType(fieldBaseType);
             }).reduce((fieldObj: any, current: any) => {
-                // FIXME read id annotation instead of hardcoding id!
-                if (current.name !== 'id') {
-                    fieldObj[current.name] = { type: current.type, description: '' };
-                }
+                const fieldType = current.type;
+                fieldObj[current.name] = { type: getNullableType(fieldType), description: '' };
```

Comment:
You only want to force the input field to be nullable when it is the primary key field.

Reply:
The same object is being used in filtering so id cannot be not null

Reply:
Should we have a separate object for filtering and mutations then?

Reply:
It can be added later when building use cases
I want to start simple with this and continue extending it with docs etc.